### DISCUSSION TITLE
any method to disablesourcemap?

### DISCUSSION THREAD

**Discussion by alanhe421** [coder/code-server#6126] [Q&A] [answered]
I found code-server cli have no param like --disable-source-map

**Comment by code-asher** [comment, marked as answer]
Yeah source maps are non-optional at the moment.

Is there any downside to them though?  They make errors easier to 
debug and I believe they are only loaded once you open dev tools 
so they should have no impact in production.

**Reply by alanhe421** [reply]
thank you.
